articles/synapse-analytics/sql/develop-tables-external-tables.md

```diff
@@ -181,6 +181,8 @@ CREATE EXTERNAL DATA SOURCE SqlOnDemandDemo WITH (
     CREDENTIAL = sqlondemand
 );
 ```
+> [!NOTE]
+> The SQL users needs to have proper permissions on database scoped credentials to access the data source in Azure Synapse Analytics Serverless SQL Pool. [GRANT Database Scoped Credential Permissions](https://docs.microsoft.com/en-us/sql/t-sql/statements/grant-database-scoped-credential-transact-sql?view=sql-server-ver15&viewFallbackFrom=azure-sqldw-latest).
```

Comment:
You might reference this page: https://docs.microsoft.com/en-us/azure/synapse-analytics/sql/develop-storage-files-overview?tabs=impersonation#permissions

Reply:
I agree with @jovanpop-msft this page has more relevant explanations.
Also remove en-us from the url when modifying due to localization.

Reply:
Changed applied